python27-sys/build.rs

```diff
@@ -1,49 +1,42 @@
-//[[[cog
-//import os
-//import cog
-//HERE = os.path.dirname(cog.inFile)
-//for line in open(os.path.join(HERE, '../python3-sys/build.rs')):
-//    if line.strip():
-//        cog.out(line)
-//    else:
-//        cog.out("// ~~~~~~~~~~ generated file, modify `python3-sys/build.rs` ~~~~~~~~~~" + line)
-//]]]
+// THIS FILE IS GENERATED FROM python3-sys/build.rs
```

Comment:
Ah, it turns out not to be true in that case

Reply:
Ah sorry, I see it's on purpose actually.

Reply:
I replaced the use of `cog` as it seemed superfluous for what it was doing, and the way the generated file annotations were inserted was causing conflicts with rustfmt.